.gitignore

```diff
@@ -4,14 +4,14 @@
 .prettierignore
 babel.config.js
 beemo-error.log
-build
-build-es5
+#build
```

Comment:
Let's not add the build files.

Reply:
Sorry! that wasn't supposed to be part of this PR; I was building so I could test it

Reply:
All good. I look forward to trying the next iteration.